apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/GenuineCheckCancelModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Button, Flex, Popin, Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+
+export type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  onSkip: () => void;
+};
+
+const GenuineCheckCancelModal = ({ isOpen, onClose, onSkip }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Popin position="relative" isOpen={isOpen}>
+      <Flex
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="space-between"
+        height="100%"
+        padding="40px"
+      >
+        <Text variant="h2" fontSize="20px" color="palette.warning.c80">
```

Comment:
With the design system normally you are not supposed to explicitely set the `fontSize` prop, it's handled by the `variant` prop.
Another thing: normally you can skip the "palette" keyword in the color prop -> `color="warning.c80"`